pytest.ini

```diff
@@ -1,4 +1,5 @@
 [pytest]
+addopts = --cov=powersimdata
```

Comment:
Is this supposed to be `adopts`? Or is it `add` `opts`?

Reply:
[reference](https://docs.pytest.org/en/6.2.x/reference.html#confval-addopts)